core/modules/user/user.install

```diff
@@ -1229,6 +1229,17 @@ function user_update_1018() {
   $config->save();
 }
 
+/**
+ * Explicitly set the "user_admin_role" attribute to "0" if it's missing.
```

Comment:
Maybe not important, but this is the text the admin user sees after running the update, right?
Maybe use "disabled" here instead of the ambiguous "0"? Just a suggestion.

Reply:
This is exactly what we're doing though 🤷 ...perhaps we could be more descriptive(?) ...perhaps something like this instead?:

```suggestion
 * Ensure that the admin role setting is disabled, by explicitly setting the
 * value of the "user_admin_role" attribute to "0" if it's missing.
```

Reply:
> This is exactly what we're doing though

That's correct. Your alternate suggestions seems a bit wordy to me. So maybe leave it as-is? Whatever you decide.

Reply:
Thanks @indigoxela 🙏 ...I'll wait for a few more people to review, and see what they think. I'll update this accordingly then.

Reply:
I've reverted this change now.